src/views/Browser.vue

```diff
@@ -700,7 +700,14 @@ export default {
         .map(word => word.charAt(0).toUpperCase() + word.slice(1))
         .join(" ");
     },
-    ...mapMutations(["setTitle", "setFilterSummary", "setFilterDescription"])
+    ...mapMutations(["setTitle", "setFilterSummary", "setFilterDescription"]),
+    handleChipClicked(termName) {
+      termName = termName.replace(/[\(\)\[\]]/g, "")
+      if (this.searchSettings.query !== null && this.searchSettings.query !== '') {
+        this.searchSettings.query += `, ${termName}`
```

Comment:
Here, instead of editing the query, you could push the term into the `searchSettings.annotations` array.